### ISSUE TITLE
ue4-build-prerequisites: installation of windows-sdk-version-2004-windbg fails (ue4-docker 0.0.75 / windows server)

### ISSUE THREAD

**Issue by arisona** [adamrehn/ue4-docker#122]
Output of the `ue4-docker info` command:

```
ue4-docker version:         0.0.75 (latest available version is 0.0.75)
Operating system:           Windows Server 2019 Standard Version 1809 (OS Build 17763.1637)
Docker daemon version:      19.03.14
NVIDIA Docker supported:    No
Maximum image size:         800GB
Available disk space:       947.36 GiB
Total system memory:        16 GiB physical, 18.37 GiB virtual
Number of processors:       4 physical, 4 logical
```

Additional details:

- Are you accessing the network through a proxy server? **No**

Installing windows-sdk-version-2004-windbg fails with latest ue4-docker version:

```
Chocolatey v0.10.15
Installing the following packages:
windows-sdk-10-version-2004-windbg
By installing you accept licenses for the packages.
Progress: Downloading windows-sdk-10-version-2004-windbg 10.0.19041.0... 100%

windows-sdk-10-version-2004-windbg v10.0.19041.0 [Approved]
windows-sdk-10-version-2004-windbg package files install completed. Performing other installation steps.
Downloading windows-sdk-10-version-2004-windbg
  from 'https://download.microsoft.com/download/1/c/3/1c3d5161-d9e9-4e4b-9b43-b70fe8be268c/windowssdk/winsdksetup.exe'
Progress: 100% - Completed download of C:\Users\ContainerAdministrator\AppData\Local\Temp\chocolatey\windows-sdk-10-version-2004-windbg\10.0.19041.0\winsdksetup.exe (1.29 MB).
Download of winsdksetup.exe (1.29 MB) completed.
Hashes match.
Installing windows-sdk-10-version-2004-windbg...
ERROR: Running ["C:\Users\ContainerAdministrator\AppData\Local\Temp\chocolatey\windows-sdk-10-version-2004-windbg\10.0.19041.0\winsdksetup.exe" /Features OptionId.WindowsDesktopDebuggers /Quiet /NoRestart /Log "C:\Users\ContainerAdministrator\AppData\Local\Temp\chocolatey\windows-sdk-10-version-2004-windbg_a8821954-a57b-4525-a3a7-3b5368378587.log" ] was not successful. Exit code was '-2146889721'. See log for possible error messages.
The install of windows-sdk-10-version-2004-windbg was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\windows-sdk-10-version-2004-windbg\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - windows-sdk-10-version-2004-windbg (exited -2146889721) - Error while running 'C:\ProgramData\chocolatey\lib\windows-sdk-10-version-2004-windbg\tools\chocolateyinstall.ps1'.
 See log for details.
The command 'cmd /S /C choco install -y windows-sdk-10-version-2004-windbg && echo. && echo.RUN directive complete. Docker will now commit the filesystem layer to disk. && echo.Note that for large filesystem layers this can take quite some time. && echo.Performing filesystem layer commit... && echo.' returned a non-zero code: 2148077575
[ue4-docker build] Error: failed to build image "adamrehn/ue4-build-prerequisites:ltsc2019".
```

Attaching the install log, seems to be a hash mismatch (again):

[windows-sdk-10-version-2004-windbg_install_failure.log](https://github.com/adamrehn/ue4-docker/files/5734704/windows-sdk-10-version-2004-windbg_install_failure.log)

Could also be a temporary network issue, will try again later.

**Comment by arisona** [comment]
Retry:  Keeps failing.

**Comment by arisona** [comment]
Some additional insights:
- Works on my Windows 10 machine
- On Windows Server, also fails outside docker environment, i.e. same error why I run from command line

**Comment by adamrehn** [comment]
Interesting. Although the Chocolatey package for `windows-sdk-10-version-2004-windbg` hasn't changed in months, I'm now seeing installation failures across all versions of Windows Server (1809, 1903, 1909, 2004, 20H2) when run through my testing harness. On the other hand, I'm seeing installation success across all versions for the `windows-sdk-10-version-1809-windbg` package, so I've switched over to that in commit a13207f. I'll test the fix to make sure it plays nicely with building the Unreal Engine before I tag a new release with the changes.

**Comment by adamrehn** [comment]
Okay, the test builds are all passing, so that seems to have fixed it. The fix is now live in ue4-docker version 0.0.76.
